core/scss/core/flex-grid/_flex-grid.scss

```diff
@@ -0,0 +1,140 @@
+@charset "UTF-8";
+
+//
+// Flex Grid
+//
+// Markup: ../../templates/core/flex-grid/flex-grid.twig
+//
+// Style guide: Core.FlexGrid
+
+// The container helper.
+.flex-container {
+  @include flex-container;
+
+  &--collapse {
+    @include margin(0 null);
+
+    @include grid-media('sm') {
+      @include margin(0 null);
+    }
+
+    @include grid-media('md') {
+      @include margin(0 null);
+    }
+
+    @include grid-media('lg') {
+      @include margin(0 null);
+    }
+
+    @include grid-media('xl') {
+      @include margin(0 null);
+    }
+
+    @include grid-media('2xl') {
+      @include margin(0 null);
+    }
+
+  }
+
+  &--row-gap {
+    > * {
+      @include grid-media('2xl') {
+        @include margin(0 0 map-get($modular-spacing, '2xl') 0);
```

Comment:
Looks like we either need to update the variable file for modular-spacing, or change $modular-spacing to $grid-gutter.